src/Maker/MakeResetPassword.php

```diff
@@ -0,0 +1,277 @@
+                'from_email' => $input->getArgument('from-email-address'),
+                'from_email_name' => $input->getArgument('from-email-name'),
+                'email_getter' => $input->getArgument('email-getter'),
+            ]
+        );
+
+        $generator->generateClass(
+            $requestClassNameDetails->getFullName(),
+            'resetPassword/ResetPasswordRequest.tpl.php',
+            [
+                'repository_class_name' => $repositoryClassNameDetails->getFullName(),
+                'user_full_class_name' => $userClassNameDetails->getFullName(),
+            ]
+        );
+
+        $generator->generateClass(
+            $repositoryClassNameDetails->getFullName(),
+            'resetPassword/ResetPasswordRequestRepository.tpl.php',
+            [
+                'request_class_full_name' => $requestClassNameDetails->getFullName(),
+                'request_class_name' => $requestClassNameDetails->getShortName(),
+            ]
+        );
+
+        if (!$this->fileManager->fileExists($path = 'config/packages/reset_password.yaml')) {
+            throw new RuntimeCommandException(sprintf('The file "%s" does not exist. This command needs that file to accurately build the reset password config.', $path));
+        }
+
+        $manipulator = new YamlSourceManipulator($this->fileManager->getFileContents($path));
+        $data = $manipulator->getData();
+
+        $data['symfonycasts_reset_password'] = ['request_password_repository' => $repositoryClassNameDetails->getFullName()];
+
+        $manipulator->setData($data);
+
+        $generator->dumpFile($path, $manipulator->getContents());
+
+        $generator->generateClass(
+            $requestFormTypeClassNameDetails->getFullName(),
+            'resetPassword/ResetPasswordRequestFormType.tpl.php',
+            [
+                'email_field' => $input->getArgument('email-property-name'),
+            ]
+        );
+
+        $generator->generateClass(
+            $changePasswordFormTypeClassNameDetails->getFullName(),
+            'resetPassword/ChangePasswordFormType.tpl.php'
+        );
+
+        $generator->generateTemplate(
+            'reset_password/check_email.html.twig',
+            'resetPassword/twig_check_email.tpl.php'
+        );
+
+        $generator->generateTemplate(
+            'reset_password/email.html.twig',
+            'resetPassword/twig_email.tpl.php'
+        );
+
+        $generator->generateTemplate(
+            'reset_password/request.html.twig',
+            'resetPassword/twig_request.tpl.php',
+            [
+                'email_field' => $input->getArgument('email-property-name'),
+            ]
+        );
+
+        $generator->generateTemplate(
+            'reset_password/reset.html.twig',
+            'resetPassword/twig_reset.tpl.php'
+        );
+
+        $generator->writeChanges();
+
+        $this->successMessage($input, $io, $controllerClassNameDetails->getFullName());
```

Comment:
We have a semi-standardized format for the success message - you can see an example here: https://github.com/symfony/maker-bundle/blob/master/src/Maker/MakeCrud.php#L206-L208

It should include something like.

```
Next:
  1) Run "php bin/console make:migration" to generate a migration for the new "ResetPasswordRequest" entity
  2) Review and customize the templates in `templates/reset_password`.
  3) Make sure your MAILER_DSN env var has the correct settings

Then go to "/reset-password" and enjoy!
```

I don't think we need any more details (that I can think of)

Reply:
WIP

Reply:
done